scripts/is_python_module_installed.py

```diff
@@ -10,11 +10,11 @@
 import pkg_resources
 
```

Comment:
Use of pkg_resources is discouraged in favor of importlib.resources, importlib.metadata, and their backports (resources, metadata). Please consider using those libraries instead of pkg_resources.

Reply:
@jofemodo this looks like a question for you to respond to.

Reply:
You created this script, @riban-bw  a few weeks ago ;-)
BTW, i moved it to a better place, in "sbin".

Reply:
LOL - I had a suspicion I did but wasn't at my workstation to validate. I will take a look.